strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/configuration/security/ldap/LdapAuthenticatorConfigurationControllerTest.java

```diff
@@ -447,10 +445,11 @@ private LdapConfigurationTestForm getLdapConfigurationTestForm()
 
         @Primary
         @Bean
-        public HazelcastInstanceId hazelcastInstanceIdLacct() {
+        public HazelcastInstanceId hazelcastInstanceIdLacct()
+        {
             return new HazelcastInstanceId("LdapAuthenticatorConfigurationControllerTest-hazelcast-instance");
         }
 
     }
 
-}
+}
```

Comment:
Please, add a new line at the end of this file. Thanks!

Reply:
Done!

Reply:
Thanks! ;)